endpoints/openrtb2/auction.go

```diff
@@ -1575,18 +1575,21 @@ func (deps *endpointDeps) parseBidExt(req *openrtb_ext.RequestWrapper) error {
 }
 
 func (deps *endpointDeps) validateAliases(aliases map[string]string) error {
-	for alias, coreBidder := range aliases {
-		if _, isCoreBidderDisabled := deps.disabledBidders[coreBidder]; isCoreBidderDisabled {
-			return fmt.Errorf("request.ext.prebid.aliases.%s refers to disabled bidder: %s", alias, coreBidder)
+	for alias, bidderName := range aliases {
+		normalisedBidderName, _ := openrtb_ext.NormalizeBidderName(bidderName)
+		coreBidderName := normalisedBidderName.String()
+		if _, isCoreBidderDisabled := deps.disabledBidders[coreBidderName]; isCoreBidderDisabled {
+			return fmt.Errorf("request.ext.prebid.aliases.%s refers to disabled bidder: %s", alias, bidderName)
```

Comment:
Why was this changed? I think validateAliases were changed in - https://github.com/prebid/prebid-server/pull/3123/files#diff-23485c351c14cdca5e337d1c05105bd3a9b7308b017589654f0854ff4f9a7ff1 PR.

Reply:
this happened because base PR PBS-1254-1 which  added changes for validateAliases was merged and deleted. Due to which commits from PBS-1254-1 got added in PBS-1257